docs/DATA_FORMATS_OUTPUT.md

```diff
@@ -13,6 +13,7 @@ plugins.
 1. [ServiceNow Metrics](/plugins/serializers/nowmetric)
 1. [SplunkMetric](/plugins/serializers/splunkmetric)
 1. [Wavefront](/plugins/serializers/wavefront)
+1. [MessagePack](/plugins/serializers/msgpack)
```

Comment:
Would you please order alphabetically!? I think the file is organized like this.

Reply:
Ack. I didn't realize it because of the first line.